components/ILIAS/Authentication/classes/class.ilObjAuthSettingsGUI.php

```diff
@@ -850,6 +850,16 @@ public function executeCommand(): void
                 $this->ctrl->forwardCommand($lpe);
                 break;
 
+            case 'ilauthlogoutpageeditorgui':
```

Comment:
Please use the class-constant instead (and normalize this and the class in the switch statement by using `strtolower`).

Reply:
done